### PR TITLE
Run axe in jest, with failing builds

### PR DESCRIPTION
**Pull request by nickcolley**
See https://github.com/nickcolley/jest-axe for source code for the module.

Timings, before:
```
Test Suites: 21 passed, 21 total
Tests:       1 skipped, 222 passed, 223 total
Snapshots:   25 passed, 25 total
Time:        3.673s
```

Timings, after:
```
Test Suites: 21 passed, 21 total
Tests:       1 skipped, 243 passed, 244 total
Snapshots:   25 passed, 25 total
Time:        3.997s
```

screen shot 2018-02-12 at 18 50 55

screen shot 2018-02-12 at 18 26 16

This work is based on the work on https://github.com/alphagov/govuk_publishing_components 's interaction with aXe

### CODE REVIEW

File: src/components/table/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ More information about when to use table can be found on [GOV.UK Design System](
 
         <tr class="govuk-c-table__row">
 
-          <td class="govuk-c-table__cell" scope="row">January</td>
+          <td class="govuk-c-table__cell">January</td>
```
Comment (nickcolley) [inline src/components/table/README.md]: Can I get a review on this change? I'm not super savvy on table semantics.

https://dequeuniversity.com/rules/axe/1.1/scope

**Comment by nickcolley** [comment]
Talked through this PR with @igloosi we agreed we should remove the table issue from this PR and raise an issue to fix it.

**Comment by igloosi** [comment]
nicely explained by @nickcolley  how it works.  would love to have @alex-ju @hannalaakso and @36degrees to have a look as well

File: package.json [hunk 2]
```diff
@@ -54,6 +53,7 @@
     "gulp-to-markdown": "^1.0.0",
     "gulp-uglify": "^3.0.0",
     "jest": "^22.1.4",
+    "jest-axe": "^2.1.0",
```
Comment (alex-ju) [inline package.json]: shall we use 2.1.1 😄 ?

Comment (nickcolley) [reply package.json]: Updated

**Comment by hannalaakso** [comment]
Looks great 🎉  It caught an image without `alt` tag and a checkboxes with empty `name` attributes.

**Pull request merged**